### PR TITLE
Added quizMe.cofee for quizzes during the associates meetings

### PR DESCRIPTION
**Pull request by Cpiely**
Still features to be added but will wait until LANBotX to add them. Fully functional for the needs I currently have.

### CODE REVIEW

**Comment by stevendiaz** [comment]
+300 lines lets do it

**Comment by arrdem** [comment]
Smells like Java. You wrote it you own it you deal with it.

File: scripts/quizMe.coffee [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+		storageLoaded()
+
+		@curCandidates = []
+		@activeQuiz = false
+
+	checkPermission: (msg) ->
+		if @admins.length == 0 || msg.message.user.name in @admins
+			return true
+		else
+			msg.send "You can't do this bitch"
```
Comment (spxrogers) [inline scripts/quizMe.coffee]: error messages

**Comment by stevendiaz** [comment]
@dwetterau What do you mean by impossible?

**Comment by dwetterau** [comment]
`else if (x <= 2 && x >= 4)` is impossible.

**Pull request merged**